src/main/java/org/testng/TestRunner.java

```diff
@@ -295,6 +293,22 @@ private void init(
     init();
   }
 
+  /**
+   * Returns all packages to use for the current test. This includes the test from the test
+   * suite. Never returns null.
+   */
+  private List<XmlPackage> getAllPackages()
```

Comment:
Styling issue:

The codebase follows this styling 
`private List<XmlPackage> getAllPackages() {`

Also please include `{}` for all `if` conditions.